pkg/storage/cstor.go

```diff
@@ -72,11 +70,14 @@ func GetCstorPools(c *client.K8sClient, pools []string) ([]metav1.TableColumnDef
 
 // DescribeCstorPool method runs info command and make call to DisplayPoolInfo to display the results
 func DescribeCstorPool(c *client.K8sClient, poolName string) error {
-	poolInfo, err := c.GetCSPI(poolName)
+	pools, err := c.GetCSPIs([]string{poolName}, "")
```

Comment:
Why do we need this change?

Reply:
This one will work agnostic of the `c.Ns` value.